authors/urls.py

```diff
@@ -29,4 +31,7 @@
     path('api/', include(('authors.apps.profiles.urls', 'profiles'), namespace='profiles')),
     path('api/', include('authors.apps.authentication.urls')),
     path('api/', include('authors.apps.rating.urls')),
+    path('api/', include('authors.apps.notifications_app.urls')),
+    path('notifications/', include(notifications.urls, namespace='notifications')),
```

Comment:
if you do this..then the notifications route won't be accessed on the api/ route
You can have it as 
 **_`path('api/notifications/', include(notifications.urls, namespace='notifications')),`_**